gatherer/pom.xml

```diff
@@ -96,7 +96,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>${jackson.xml.version}</version>
+            <version>${jackson.xml.version}.2</version><!-- NB: 4th digit release! -->
```

Comment:
@slitsche this is quite obscure, but I don't have better ideas at the moment. :-/